<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterAccount</class>
 <widget class="QDialog" name="RegisterAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass Confirm</string>
    </property>
    <property name="buddy">
     <cstring>label_5</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass:</string>
    </property>
    <property name="buddy">
     <cstring>label_4</cstring>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtPass">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtPassConfirm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonFinish">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>finish</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtUserName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SSN</string>
    </property>
    <property name="buddy">
     <cstring>label_4</cstring>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtSSN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Account Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtAccountName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
